packages/openneuro-components/stories/icon/Icon.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react'
+
+import { Icon, IconProps } from './Icon'
+
+import orcidIcon from '../assets/orcid_24x24.png'
+
+export default {
+  title: 'Components/Icon',
+  component: Icon,
+  argTypes: {
+    backgroundColor: { control: 'color' },
+    color: { control: 'color' },
+  },
+} as Meta
+
+const Template: Story<IconProps> = args => <Icon {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  label: 'Default',
+  icon: 'fab fa-google',
+}
+
+export const ExampleTwo = Template.bind({})
+ExampleTwo.args = {
+  icon: 'fab fa-google',
+  color: 'red',
+}
+
+export const IconImage = Template.bind({})
+IconImage.args = {
+  imgSrc: orcidIcon,
+  iconSize: '23px',
+}
+
+export const IconWithBackgroundColor = Template.bind({})
+IconWithBackgroundColor.args = {
+  icon: 'fab fa-google',
+  iconSize: '12px',
+  color: '#fff',
+  backgroundColor: '#555',
+}
```

Comment:
I think this could be organized better, either just removing the stories part of the file paths or moving the components to the top level and keeping stories in a stories directory. Making it a separate directory makes it easier to not include the stories in the final bundle but as long as the pattern to identify stories is simple, they be mixed in which might be preferred just to keep you from digging when working on a component and the related stories.

Reply:
To be consistent, how about this? openneuro-components/src/icon/Icon.tsx and openneuro-components/src/icon/Icon.stories.tsx

Reply:
Should this be ?

openneuro-components/src/icon/Icon.tsx and openneuro-components/**stories**/icon/Icon.stories.tsx

Reply:
It could be but I don't think it needs to be. We can add an exclude pattern for .stories.* files when compiling it. I like having tests next to the real component, just generally makes it easier to work on both at the same time with less hunting around the repo. What do you think?